adlchecker/src/main/java/com/nedap/archie/adlchecker/TerminologyContentGenerator.java

```diff
@@ -0,0 +1,167 @@
+package com.nedap.archie.adlchecker;
+
+import com.nedap.archie.adlparser.ADLParser;
+import com.nedap.archie.aom.Archetype;
+import com.nedap.archie.aom.CAttribute;
+import com.nedap.archie.aom.CObject;
+import com.nedap.archie.aom.terminology.ArchetypeTerm;
+import com.nedap.archie.aom.utils.AOMUtils;
+import com.nedap.archie.rminfo.MetaModels;
+
+import java.io.IOException;
+import java.util.Comparator;
+import java.util.List;
+import java.util.ArrayList;
+import java.util.Map;
+import java.util.Stack;
+import java.util.TreeMap;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * If you create an archetype, generates a line of empty terminology for every missing term in every language.
+ * Adds the comments as default text
+ */
+public class TerminologyContentGenerator {
+
+    private MetaModels models;
+    Pattern commentPattern = Pattern.compile(".*\\[(?<idcode>id[0-9]+)(,|\\]).*--(?<comment>.*)");
+
+    public TerminologyContentGenerator(MetaModels models) {
+        this.models = models;
+    }
+
+    public Archetype addTerms(String adlContent) {
+        ADLParser parser = new ADLParser();
+        try {
+            Archetype archetype = parser.parse(adlContent);
+            models.selectModel(archetype);
+
+            //TODO: just run the CodeValidation in the context of an archetype repository, and process the error messages?
+            if(parser.getErrors().hasErrors()) {
+                throw new RuntimeException("parse errors!" + parser.getErrors().toString());
+            }
+            Archetype resultingArchetype = parser.parse(adlContent); //instantiate twice
```

Comment:
Need two copies. archetype.clone() would have been ok as well.

I think this class can be removed in the future since we now have a decent editor tool, making these kind of basic command-line tools obsolete